File: Day 2/Ques 4.cpp
//question Link: https://leetcode.com/problems/remove-duplicates-from-sorted-array/
//solution:

class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        if(nums.size()==0){
            return 0;
        }
        for(int i=0;i<nums.size()-1;i++){
            if(nums[i]==nums[i+1]){
                nums.erase(nums.begin()+i);
                i--;
            }
            else{
                continue;
            }
        }
        return nums.size();
    }
};

File: Day13/Ques4.py
#Question link: https://leetcode.com/problems/truncate-sentence/

class Solution:
    def truncateSentence(self, s: str, k: int) -> str:
        st=s.split()
        st=st[:k]
        s=""
        for i in st:
            s+=i+" "
        return s[:-1]
